lms/djangoapps/instructor/views/api.py

```diff
@@ -2448,3 +2453,443 @@ def mark_student_can_skip_entrance_exam(request, course_id):  # pylint: disable=
         'message': message,
     }
     return JsonResponse(response_payload)
+
```

Comment:
This file is getting super crazy long. I wonder if it's worth putting some of these methods into a helper file of some sort.